djgeojson/views.py

```diff
@@ -88,7 +88,23 @@ def get_queryset(self):
         """
         Inspired by Glen Roberton's django-geojson-tiles view
         """
-        self.z, self.x, self.y = map(int, self.args[:3])
+        try:
+            self.z, self.x, self.y = map(int, self.args[:3])
+        except AttributeError:
+            # let's try to get these as kwargs
+            self.z = self.kwargs.get("z", None)
+            self.x = self.kwargs.get("x", None)
+            self.y = self.kwargs.get("y", None)
```

Comment:
I don't think so, but I'll add tests and a conversion step to guarantee that. The Django URL system should capture that registered URL only if they are already integers.